src/legacy/status_im/subs/root.cljs

```diff
@@ -22,7 +22,6 @@
 (reg-root-key-sub :peers-count :peers-count)
 (reg-root-key-sub :peers-summary :peers-summary)
 (reg-root-key-sub :web3-node-version :web3-node-version)
-(reg-root-key-sub :alert-banners :alert-banners)
```

Comment:
could you elaborate?

Reply:
It already exists here
https://github.com/status-im/status-mobile/blob/1129a9c3cbb299814506660eee7c53567bbc3073/src/status_im/subs/root.cljs#L67